elyra/pipeline/component_parameter.py

```diff
@@ -290,14 +290,19 @@ class DisableNodeCaching(ElyraProperty):
     property_description = "Disable caching to force node re-execution in the target runtime environment."
     _json_data_type = "string"
 
-    def __init__(self, selection, **kwargs):
-        self.selection = selection == "True"
+    def __init__(self, selection: Union[str, bool], **kwargs):
+        self.selection = None
+        if selection in ["True", "true", True]:
```

Comment:
Is there a reason this is not just a pure boolean type?  Does the front end only persist string values into the pipeline JSON for this field?

Reply:
You're exactly right that right now it avoids migration 😄 . Currently the frontend persists strings only, but we intend to try to change this in 3.14. I only say try because we have some other potential limitations, e.g. the placeholder value is a string. Checking for boolean types is probably unnecessary right now, but I'll probably open a PR later this week that *would* require this change, so just 'preparing' for that